src/parquet/arrow/arrow-reader-writer-test.cc

```diff
@@ -1403,6 +1403,56 @@ TEST(TestArrowReadWrite, ConvertedDateTimeTypes) {
   AssertTablesEqual(*ex_table, *result);
 }
 
+// Regression for ARROW-2802
+TEST(TestArrowReadWrite, CoerceTimestampsAndSupportDeprecatedInt96) {
+  using namespace ::arrow;
```

Comment:
Please list the types you use explicitly here, e.g. `using ::arrow::TimestampType`.